.github/workflows/wsl.yaml

```diff
@@ -22,10 +22,24 @@ jobs:
           channel: 'stable'
           flutter-version: '3.10.x'
 
+      - name: Update pubspec.lock
+        working-directory: packages\ubuntu_wsl_setup\
+        shell: bash
+        run: flutter pub upgrade
+
+      - name: Create PR
+        uses: peter-evans/create-pull-request@v5
+        with:
+          add-paths: packages/ubuntu_wsl_setup/pubspec.lock
```

Comment:
should we also have something like `add-paths: **/*.mocks.dart` for mocks and `add-paths: **/l10n/*.dart` for l10n?

- https://github.com/canonical/ubuntu-desktop-installer/pull/2031
- https://github.com/canonical/ubuntu-desktop-installer/pull/2032
- https://github.com/canonical/ubuntu-desktop-installer/pull/2033

Reply:
Yeah! I think we should have something like that 😜